LICENSE.md

```diff
@@ -73,19 +79,19 @@ The User may examine the Mod's source code.
 The Author reserves the right to distribute the Mod.  
 The Author may designate a person or organization as a Distributor.  
 A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
+The Author may require that a Distributor provides a direct link to the Mod.  
 The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author may revoke distribution rights from any Distributor without prior notice.  
 
 6. Right to contribute
 --------
 
-The User may "fork" the Codebase.  
+The User may fork the Mod or any Software associated.  
```

Comment:
No. This license intentionally avoids making any statements about what the user can or can't do with other things. Remove this, please, unless you can provide a good reason why it should be added.

Additionally, you don't fork a mod...you fork the codebase. So this change is both overly intrusive (defining what the user can/can't do outside of the licensed code) and incorrect.